foundation-models/openai/src/main/java/com/sap/ai/sdk/foundationmodels/openai/OpenAiClient.java

```diff
@@ -34,7 +34,7 @@
 @Slf4j
 @RequiredArgsConstructor(access = AccessLevel.PRIVATE)
 public final class OpenAiClient {
-  private static final String DEFAULT_API_VERSION = "2024-02-01";
+  private static final String DEFAULT_API_VERSION = "2024-10-21";
```

Comment:
(Comment)

I would not merge until actual API changes are being considered in the Java classes.

Reply:
To be done with AI/ai-sdk-java-backlog#118 then